app/views/offers/index.html.erb

```diff
@@ -36,9 +36,6 @@
             <span class="caret"></span>
           </a>
           <ul class="dropdown-menu" role="menu">
-            <li>
-              <%= link_to "× #{t 'global.all'}", dashboard_offers_path %>
-            </li>
```

Comment:
I'd leave this link, as we also have in inquiries list. It's useful when you want to remove current search by category.